cns/ipampoolmonitor/ipampoolmonitor.go

```diff
@@ -100,8 +102,8 @@ func (pm *CNSIPAMPoolMonitor) Reconcile() error {
 }
 
 func (pm *CNSIPAMPoolMonitor) increasePoolSize() error {
-	pm.mu.Lock()
 	defer pm.mu.Unlock()
+	pm.mu.Lock()
```

Comment:
Curious, why the flip?

Reply:
No reason in particular, just coding practice to add a defer unlock API before Lock. 

---
In reply to: [585721003](https://github.com/Azure/azure-container-networking/pull/809#discussion_r585721003) [](ancestors = 585721003)